core/src/test/java/org/testcontainers/utility/AuthenticatedImagePullTest.java

```diff
@@ -90,6 +91,7 @@ public void removeImageFromLocalDocker() {
         // remove the image tag from local docker so that it must be pulled before use
         try {
             client.removeImageCmd(testImageName).withForce(true).exec();
+            LocalImagesCacheAccessor.clearCache();
```

Comment:
Should this be after the `catch`?
I'm not sure that much could go wrong from clearing the cache too many times, but if we fail to then odd things might happen.